Makefile

```diff
@@ -14,7 +14,7 @@ RUST_2018_IDIOMS = -D bare-trait-objects \
                    -D ellipsis-inclusive-range-patterns \
                    -D unused-extern-crates
 
-DENY = -D warnings -D future-incompatible -D unused ${RUST_2018_IDIOMS}
+DENY = -D warnings -D future-incompatible -D unused -D rust_2018_idioms -D rust_2018_compatibility -D nonstandard_style
```

Comment:
You can remove ```RUST_2018_IDIOMS``` variable, I believe.

Reply:
Sorry; I missed that somehow.  Resolved.